compiler/ballerina-lang/src/main/resources/grammar/BallerinaLexer.g4

```diff
@@ -648,6 +655,11 @@ ExpressionStart
     :   '{{'
     ;
 
+fragment
+InterpolationExpressionStart
```

Comment:
since this is a leaf node, better to make it all-caps. may be make it a bit short like `INTERPOLATION_START`

Reply:
I followed the same convention used for most of the other fragments including leaf nodes (Eg: ExpressionStart, PaddingChar...).

Reply:
The convention is to use all-caps for terminal symbols and camel case is for non-terminal rules.

Reply:
Ack. I will update the PR accordingly.